src/components/preview/preview.scss

```diff
@@ -0,0 +1,15 @@
+.carbon-preview {
+  &--placeholder {
+    opacity: 0.6;
+    height: 24px;
+    width: 100%;
+    background: #CCCCCC;
```

Comment:
Where did this color come from. Do we have a colour variable for this or something similar. May need UX input

Reply:
👍Good point. Changed it for  ```background: $grey-dark-blue-25;```